backend/queries/events.js

```diff
@@ -49,8 +51,16 @@ const getAllEvents = async (vName, topic, instructor, upcoming, past) => {
       events.location, 
       events.instructor, 
       events.number_of_volunteers, 
-      cohorts.cohort
-    ORDER BY event_start DESC
+      cohorts.cohort,
+      cohorts.cohort_id
+    ORDER BY (
```

Comment:
@Briahana-Mauge I'm putting back this form you code, that was a good way to separate between past and upcoming events